### PR TITLE
Add option to force staging

### PR DESCRIPTION
**Pull request by jeffcarbs**
~~Disclaimer: I couldn't get the tests running on my machine so I wasn't able to add tests.~~ Got it running and added test 👍

### CODE REVIEW

File: core/task.go [hunk 1]
```diff
@@ -12,8 +12,9 @@ type Task struct {
 	Command string
 	Files   string
 	Fix     struct {
-		Command string
-		Output  string
+		Command   string
+		Output    string
+		Autostage string
```
Comment (nikhilmat) [inline core/task.go]: should we make this a `bool` instead?

**Comment by jeffcarbs** [comment]
@nikhilmat - good point re: using `bool. For some reason I didn't think you couldn't represent a bool in a YAML file. Again, I haven't actually tested this since I couldn't get go up and running.

**Comment by nikhilmat** [comment]
@jeffcarbs do you want help getting it setup? its pretty straightforward as long as you adhere to the GOPATH conventions. I think we want tests to verify before merging

**Comment by jeffcarbs** [comment]
@zp-alfred review requested

@nikhilmat - added some tests. Also fixed some of the assertion messages that I noticed were incorrect (e.g. test expects an error but message says "there is no error").

**Comment by nikhilmat** [comment]
Thanks @jeffcarbs!

@zp-alfred accept

**Comment by zp-alfred** [comment]
Accepted by nikhilmat!

**Pull request merged**